providers/pg_database.rb

```diff
@@ -12,6 +12,7 @@ def whyrun_supported?
 action :create do
 
   project_name = node['enterprise']['name']
+  connection_string = "--host #{new_resource.host} --username #{new_resource.username} --password #{new_resource.password}" unless node[project_name]['postgresql']['enable']
```

Comment:
I'm worried that the assumptions we're making here may not necessarily hold up in the future. We're saying here that we're going to pass a host/username/password if postgres is not enabled.

What if for some reason I was using user/pass auth with an enabled postgres? What if I had some weird networking compliance rules where I have to use a host instead of a socket?

I think maybe just continuing the pattern used by the `owner` property, where we add to the connection string if the property is present, would be more maintainable in the long run.